packages/rancher-logging/4.10/generated-changes/overlay/templates/loggings/rke2/configmap.yaml

```diff
@@ -3,6 +3,7 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: {{ .Release.Name }}-rke2
+  namespace: {{ include "logging-operator.namespace" . }}
```

Comment:
Do we want to only use `{{ include "logging-operator.namespace" . }}`? If yes, line 65 needs to be changed